features/gm.ts

```diff
@@ -81,23 +87,36 @@ export class GroupMessageController implements GroupMessageGetter, GroupMessageL
         return event;
     }
 
-    async prepareGroupMessageEvent(groupAddr: PublicKey, text: string) {
+    async prepareGroupMessageEvent(groupAddr: PublicKey, message: string | Blob) {
```

Comment:
A blob is not necessarily an image. We can keep it for now. In the future, we need to change because audio and normal files are also `Blob`